restres/owner.html

```diff
@@ -0,0 +1,562 @@
+                                            <tr>   <td> 6</td> <td class="text-danger"> Occupied</td> <td> 6.30PM</td> <td> 4</td> <td> <a href="#" id="c">#67897 </a></td> </tr>
+                                            <tr>   <td> 3</td> <td class="text-danger "> Occupied</td> <td> 7.15PM</td> <td> 2</td> <td>  <a href="#" id="e">#75674 </a></td> </tr>
+                                            <tr>   <td> 2</td> <td class="text-success bg-success"> Available</td> <td>  - </td> <td> -</td> <td> <a href="#"> - </a></td> </tr>
+                                            <tr>   <td> 4</td> <td class="text-success bg-success"> Available</td> <td> - </td> <td> -</td> <td> <a href="#">-</a></td> </tr>
+                                            </tbody>
+                                        </table>
+                                    </div>
+
+                                    <div class="col-sm-5">
+                                        <table class="table table-condensed table-bordered bg-muted" id="b" style="display:none;">
+                                            <thead class="bg-danger">
+                                            <tr>  <th>Cnf. No.</th> <th>Name</th> <th>Date&Time</th> <th>Size</th>  </tr>
+                                            </thead>
+                                            <tbody>
+                                            <tr>   <td> #12345</td> <td> Ritanshi Parmar</td> <td> 12/20,6.15PM</td> <td> 3</td>  </tr>
+                                            </tbody>
+                                        </table>
+                                    </div>
+
+                                    <div class="col-sm-5">
+                                        <table class="table table-condensed table-bordered bg-muted" id="d" style="display:none;">
+                                            <thead class="bg-danger">
+                                            <tr>  <th>Cnf. No.</th> <th>Name</th> <th>Date&Time</th> <th>Size</th>  </tr>
+                                            </thead>
+                                            <tbody>
+                                            <tr>   <td> #67897</td> <td> Ritanshi Parmar</td> <td> 12/20,6.30PM</td> <td> 4</td>  </tr>
+                                            </tbody>
+                                        </table>
+                                    </div>
+
+
+                                    <div class="col-sm-5">
+                                        <table class="table table-condensed table-bordered bg-muted" id="f" style="display:none;">
+                                            <thead class="bg-danger">
+                                            <tr>  <th>Cnf. No.</th> <th>Name</th> <th>Date&Time</th> <th>Size</th>  </tr>
+                                            </thead>
+                                            <tbody>
```

Comment:
do not use tables for aligning data on screen, use only when you want to show tabular data